### PR TITLE
7554 chart axes

### PR DESCRIPTION
**Pull request by stephensprinkle**
[Clubhouse Story](https://app.clubhouse.io/augur/story/7554)

## Submitter checklist
- [x] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [x] Steps to reproduce
  - [x] Screenshot (If applicable)

## Reviewer Checklist
- [ ] Test/lint pass 
- [ ] Post merge, story marked complete

### CODE REVIEW

File: src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx [hunk 1]
```diff
@@ -306,3 +308,299 @@ function nearestCompletelyFillingOrder(price, marketDepth) {
...
+  } = options
+
+  depthChart.append('text')
+    .attr('id', 'hovered_price_label')
+
+  // create crosshairs
+  const crosshair = depthChart.append('g')
+    .attr('id', 'crosshairs')
+    .attr('class', 'line')
+    .attr('style', { display: 'none' })
```
Comment (justinbarry) [inline src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx]: Can/Should this be `.style()`?

Comment (stephensprinkle) [reply src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx]: Ah, yep, all utilizations should be migrated to `style` — will do that!

File: src/modules/market/components/market-outcome-charts--candlestick/market-outcome-charts--candlestick.jsx [hunk 2]
```diff
@@ -34,361 +32,523 @@ export default class MarketOutcomeCandlestick extends Component {
...
+    .attr('width', drawParams.drawableWidth)
+    .attr('height', drawParams.containerHeight)
+    .on('mousemove', () => updateHoveredPrice(drawParams.yScale.invert(d3.mouse(d3.select('#candlestick_chart').node())[1]).toFixed(fixedPrecision)))
+    .on('mouseout', () => updateHoveredPrice(null))
+    .on('click', () => {
+      const mouse = d3.mouse(d3.select('#candlestick_chart').node())
+      const orderPrice = drawParams.yScale.invert(mouse[1]).toFixed(fixedPrecision)
+
+      if (
+        orderPrice > this.props.marketMin &&
```
[the remark this answers is not in the document]
Comment (stephensprinkle) [reply src/modules/market/components/market-outcome-charts--candlestick/market-outcome-charts--candlestick.jsx]: Bah, yeah, that needs to get fixed!

**Comment by stephensprinkle** [comment]
Addressed :sparkles:

**Comment by justinbarry** [comment]
1. Clicking on the graph without any data blows up.

**Pull request merged**